README.md

```diff
@@ -14,21 +14,25 @@ If you find any problems with the code or have any ideas on improving it, please
 
 ## Table of Contents
 
--   [Getting Started](#getting-started)
--   [High-level Architecture](#high-level-architecture)
--   [Preparing Your Environment](#preparing-your-environment)
--   [Running MLflow](#running-mlflow)
--   [Training The Model](#training-the-model)
--   [Visualizing Pipeline Results](#visualizing-pipeline-results)
--   [Deploying The Model](#deploying-the-model)
--   [Monitoring The Model](#monitoring-the-model)
--   [Production Pipelines in Amazon Web Services](#production-pipelines-in-amazon-web-services)
+- [Building Machine Learning Systems](#building-machine-learning-systems)
```

Comment:
Let's keep "Building Machine Learning Systems" and "Table of Contents" outside of the Table of Contents.

Reply:
Sure.